tooling/cli/src/mobile/android/dev.rs

```diff
@@ -112,9 +119,29 @@ fn run_dev(
   let env = env()?;
   init_dot_cargo(app, Some((&env, config)))?;
 
+  if let Some(device) = &options.device {
```

Comment:
is `options.device` defined by the user as a command line argument? or do we prompt them for available avds to choose one?

Reply:
It's provided by the user. We'll prompt if it's not set and there's multiple connected devices.

I'll also improve it to prompt for emulators if there's no connected device.